packages/launch/src/focused-launch/summary/index.tsx

```diff
@@ -326,11 +327,13 @@ const PlanStep: React.FunctionComponent< PlanStepProps > = ( {
 		? [ defaultPaidPlan, nonDefaultPaidPlan, defaultFreePlan ]
 		: [ defaultPaidPlan, defaultFreePlan ];
 
-	const allAvailablePlansProducts = useSelect( ( select ) => {
-		return allAvailablePlans.map( ( plan ) =>
-			select( PLANS_STORE ).getPlanProduct( plan?.periodAgnosticSlug, billingPeriod )
-		);
-	} );
+	const allAvailablePlansProducts = useSelect(
```

Comment:
As explained in [this comment}(https://github.com/Automattic/wp-calypso/pull/49412#pullrequestreview-582249246), when there isn't a selected plan in the launch store, `selectedPlanBillingPeriod` is `undefined`.

This causes the `allAvailablePlansProducts` to become an array of `undefined` objects.
Together with adding a fallback value of `ANNUALLY` to `selectedPlanBillingPeriod`, we should also avoid showing any plans-related UI when there are no plan products in `allAvailablePlansProducts`.

Reply:
I've extended the check on [L406-407](https://github.com/Automattic/wp-calypso/pull/49412/files#diff-bae2915b59f2e567e179b1ed48bdcb4d210fbc99442732fa6252d32ef7c2c181R406-R407) to take the Plan Products into account as well.